log/value_test.go

```diff
@@ -10,26 +10,26 @@ import (
 
 func TestValueBinding(t *testing.T) {
 	var output []interface{}
+
 	logger := log.Logger(log.LoggerFunc(func(keyvals ...interface{}) error {
 		output = keyvals
 		return nil
 	}))
-	logger = log.With(logger, "ts", log.Timestamp, "caller", log.Caller)
+	logger = log.With(logger, "ts", log.DefaultTimestamp, "caller", log.DefaultCaller)
 
-	time.Sleep(50 * time.Millisecond)
```

Comment:
True. And in reality the test just needs to validate that the function gets called from the right place, so the time value doesn't actually matter. The test can inject a function that increments a counter and check that the increment happens only on Log calls.